docs/devguide/newdashboards.asciidoc

```diff
@@ -49,10 +49,12 @@ To import the dashboards, run the `setup` command.
 The `setup` phase loads:
 
 - Index mapping template in Elasticsearch
-- Kibana dashboards
-- Machine Learning jobs (if available)
+- Kibana dashboards (where available)
+- Machine Learning (ML) jobs (where available)
+- Ingest pipelines (where available)
+- ILM policy (for Elasticsearch 6.5 and newer)
```

Comment:
Actually, not sure about the ES version here, afaik it should be >= 6.7 (supported) or >= 7.0 (default).

Reply:
Actually I'm not sure it's necessary to mention the version number at all here.

Reply:
@simitt ^^ The version info shown here is taken directly from the output of `help setup`. I'm not sure it's necessary here, though, so Ill will change the lead-in wording in this section.